src/modules/emotes/style.css

```diff
@@ -98,6 +98,12 @@ img.emoji.yt-live-chat-text-message-renderer + .bttv-global-emo-5e76d399d6581c37
   margin: -0.5rem 0;
 }
 
+div[data-test-selector='emote-button'] + span[data-a-target='chat-message-text'] span .bttv-emote-overlay,
+div.bttv-emote + .bttv-emote-overlay,
+img.emoji.yt-live-chat-text-message-renderer + .bttv-emote-overlay {
+  margin-left: -32px;
```

Comment:
how does this logic interplay today with wide emotes?

Reply:
In the 7TV V3 extension, it appears as though the emote token object has a field dedicated for overlay emotes and the tokenizer always keeps track of the last emote token, which makes interplay with wide emotes almost native since it's not relying on hard coded css values. https://github.com/SevenTV/ExtensionV3/blob/ea8f1d9e5488e14995b7bd7eaaa0d063dcd5f311/src/common/chat/Tokenizer.ts#L24-L35

I'm not sure how emotes are handled by the Bttv extension and I know both extensions use different frameworks, but maybe this information still helps?

Reply:
hard coded css values can work fine here for wide emotes too, we just need to account for them properly in the stylesheet

Reply:
<img width="768" alt="Screenshot 2023-04-01 at 02 08 40" src="https://user-images.githubusercontent.com/43322006/229258324-86bbe58e-4b71-4057-97ed-17b0ae69d375.png">

Seems like a niche use-case, worth supporting wide overlay emotes?

Reply:
I think so, because suddenly it can be used on a wide emote and become meta. I just can't know with twitch chat what will happen next

Reply:
I am personally fine with this result; it is identical to how overlays are handled in another project I help develop.

Another question to ask though is how do wide overlays behave? Example: https://7tv.app/emotes/6360b863593eb316d898eb78